libs/category/src/drivers/magento/transformers/category-response-transform.service.spec.ts

```diff
@@ -106,18 +105,16 @@ describe('DaffMagentoCategoryResponseTransformService', () => {
       expect(magentoCategoryPageConfigurationTransformerServiceSpy.transform).toHaveBeenCalledWith(completeCategory);
     });
 
-    it('should call transformMany on the magentoCategoryTransformerService', () => {
-      service.transform(completeCategory);
-
-      expect(magentoProductTransformerServiceSpy.transformMany).toHaveBeenCalledWith(completeCategory.products);
+    it('should return the same number of products it receives', () => {
+      expect(service.transform(completeCategory).products.length).toEqual(completeCategory.products.length);
     });
 
     it('should return a DaffGetCategoryResponse compiled from the other injected transformers', () => {
```

Comment:
This comment doesn't make sense any longer.

Reply:
I changed it